trans-app/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.10.15-bullseye
+
+RUN apt-get -y  update; apt-get -y install sudo build-essential nano git wget libgl1-mesa-glx
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
```

Comment:
Cài xong thì nên xoá file `requirements.txt` luôn nha sếp

Reply:
Với sao k dùng base image là slim mà lại dùng bullseye nhỉ sếp? https://medium.com/@faruk13/alpine-slim-bullseye-bookworm-noble-differences-in-docker-images-explained-d9aa6efa23ec